src/styles/index.scss

```diff
@@ -232,8 +236,22 @@ $letter-zoom-scale: 0.5;
     position: relative;
   }
 
-  .link-to-letter > a {
-    padding: 0.2em 0.8em;
+  .link-to-letter {
+    position: absolute;
+    z-index: 999999999999;
```

Comment:
LMAO, can you extract this to a constant

Reply:
also maybe instead then for the zindex thing we should store highestZ and ref to the last dragged letter and then when you reset highest you can reset the zindex of the last ref

Reply:
i feel like that might be worth doing